packages/extension-core/src/domains/signing/handler.ts

```diff
@@ -94,16 +95,22 @@ export default class SigningHandler extends ExtensionHandler {
               )
 
         if (payload.withSignedTransaction) {
-          const tx = registry.createType(
-            "Extrinsic",
-            { method: payload.method },
-            { version: payload.version }
-          )
-
-          // apply signature to the modified payload
-          tx.addSignature(payload.address, signature, payload)
-
-          signedTransaction = tx.toHex()
+          try {
+            const tx = registry.createType(
+              "Extrinsic",
+              { method: payload.method },
+              { version: payload.version }
+            )
+
+            // apply signature to the modified payload
+            tx.addSignature(payload.address, signature, payload)
```

Comment:
The v15 chain metadata instructs us to build a 66-byte signature, but the chain will reject anything other than a 65-byte signature.

We trim the first byte off of the signature to enable signing.
However, `registry` still expects to see a 66-byte signature (as its understanding is based on the v15 chain metadata).

As such, our workaround is the `try {} catch {}` around this block which will *attempt* to create the `signedTransaction` needed for ledger support, but will gracefully fall back if it cannot.